pkgs/build-support/closure-info.nix

```diff
@@ -4,7 +4,7 @@
 # "nix-store --load-db" and "nix-store --register-validity
 # --hash-given".
 
-{ stdenv, coreutils, jq }:
+{ stdenv, coreutils, jq, buildPackages }:
```

Comment:
The `coreutils` and `jq` arguments aren't needed here anymore

Reply:
Yes, I had had that thought.  I wondered though if I've fixed it in the wrong place and there should be something in the way that make-closure is invoked/included/called by its callers which would cause it to use the build-system versions of coreutils and jq without my having to indirect explicitly through buildPackages.  Maybe @Ericson2314 or @dtzWill would know?

Reply:
If we were to do
```
nativeBuildInputs = [ jq, coreutils];
```

it would indeed happen automatically, due to the very magical and aweful "`callPackage` + `mkDerivation` splicing we do. Without that, however, one needs to pick `buildPackages` by hand.

Reply:
It sounds like it would be better to stick with referring through `buildPackages`, but maybe that's my very low tolerance for magic?  I'll remove the arguments that we don't need